### ISSUE TITLE
hypoplastic hemidesmosome

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4827]
**Preferred term label:**
hypoplastic hemidesmosome

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://www.ncbi.nlm.nih.gov/pubmed/24533970

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0032449

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Hemidesmosomes (HDs) are highly specialized integrin-mediated epithelial attachment structures that make cells firmly adhere to the extracellular matrix by establishing a link between the underlying basement membrane (BM) and the internal mechanical stress-resilient keratin intermediate filament (IF) network. PMID:25487405

The HD themselves are normally structured with regard to their extracellular portions and have well developed subbasal dense plates and anchoring filaments in the lamina rara, but their intracellular attachment plates are mostly thin and frequently separated into small, triangular segments; their thickness is then about half of normal HD attachment plates. 
https://www.jidonline.org/article/S0022-202X(15)41534-9/fulltext

**Comment by pnrobinson** [comment]
adding as HP_0020117
